example-app/build.gradle

```diff
@@ -71,6 +71,7 @@ android {
 dependencies {
     // Checkout
     implementation project(':drop-in')
+//    implementation "com.adyen.checkout:drop-in:4.1.1"
```

Comment:
Is that added on purpose?

Reply:
Yeah, for when we want to make a test against the release artifacts or a test artifact in `mavenLocal()`, just have it there commented to make it easier to change =P

Reply:
👍